include/mesh/distributed_mesh.h

```diff
@@ -81,7 +81,27 @@ class DistributedMesh : public UnstructuredMesh
    * Copy and move assignment are not allowed.
```

Comment:
Gah, I noticed that this comment was being obsoleted in replicated_mesh.h, but missed it here.